.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -90,4 +99,4 @@ body:
     attributes:
       label: Relevant log output
       description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
-      render: shell
+      render: Shell
```

Comment:
Spec wants this value to be uppercase, refer:
 - https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#attributes-for-textarea
 - https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml#L6383